pie/bin/start

```diff
@@ -22,7 +22,7 @@ docker run -d -it -v $PWD:/work -w /work \
   -e CONJUR_ACCOUNT="$ACCOUNT" \
   -e CONJUR_APPLIANCE_URL="eval.conjur.org" \
   --entrypoint bash conjurinc/cli5 -c '
-    cat policy/root.yml | sed -e "$replace" | conjur policy load --replace root -;
```

Comment:
it could even just be `conjur policy load root policy/policy_5.yml`, the `cat` is unnecessary